### PR TITLE
Update AboutUs

### PR DESCRIPTION
**Pull request by JordanChua**
- Update `AboutUs.md` to include my photo and current role (#47).
- Add my photo in `doc/images/jordanchua.png`.
-  Add my skeletal PPP in `docs/team/jordanchua.md` (#49).

### CODE REVIEW

**Review by RussellDash332 — CHANGES_REQUESTED** [review]
Your PNG filename is `jordanchua.png.png`, please rename it 😄

**Comment by JordanChua** [comment]
> Your PNG filename is `jordanchua.png.png`, please rename it 😄

Updated 👍

File: docs/AboutUs.md [hunk 1]
```diff
@@ -29,11 +29,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Team Lead
 * Responsibilities: UI
 
-### Johnny Doe
+### Chua Yu Xuan Jordan 
 
-<img src="images/johndoe.png" width="200px">
```
Comment (RussellDash332) [inline docs/AboutUs.md:34]: What I meant is that `jordanchua.png.png` is the incorrect one 😅

**Review by RussellDash332 — CHANGES_REQUESTED** [review]
You should rename the image instead of the link at AboutUs 😆

**Comment by JordanChua** [comment]
> You should rename the image instead of the link at AboutUs 😆

Sorry boss i updated it

**Review by RussellDash332 — APPROVED** [review]
LGTM!

**Pull request merged**